### ISSUE TITLE
Arabic Handwritten Digits Dataset

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/ML-Crate#160]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Arabic Handwritten Digits Dataset
:red_circle: **Aim** : Create a prediction model which will predict the arabic digits correctly.
:red_circle: **Dataset** : https://www.kaggle.com/mloey1/ahdd1
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

Hello, ML-Crate contributors, this issue is only for the contribution purposes and allocated only to the participants of SWOC 2.0 Open Source Program and JWOC '22 Open Source Program.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.
- This issue is only for 'SWOC' and 'JWOC' contributors of 'ML-Crate' project.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID :
- Approach for this Project :
- What is your participant role?
    - [ ] SWOC 2.0 Participant.
    - [ ] JWOC 2022 Participant.
    - [ ] Contributor

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Pratyush-IITBHU** [comment]
Can SWOC participants take it??

**Comment by abhisheks008** [comment]
@Pratyush-IITBHU No!

**Comment by DhruvKarotra** [comment]
Full name : Dhruv Karotra
GitHub Profile Link : https://github.com/DhruvKarotra
Participant ID: NA
Approach for this Project: I will use Tensorflow and Keras using CNN to identify the arabic handwritten digits.

**Comment by abhisheks008** [comment]
@DhruvKarotra not just stick with a single algorithm/model, try to explore and implement 3-4 models and then find out the best fitted one based on their accuracy scores.

Issue assigned to you. Go ahead!

**Comment by abhisheks008** [comment]
Issue has been un-assigned from you due to inactivity.
@DhruvKarotra

**Comment by Han9128** [comment]
Name: Md Shahreyar Hannan
GitHub Profile Link: https://github.com/Han9128
Participant Id:  NA
Approach for this Project: I will implement TensorFlow and Keras to make a neural network to identify the digits correctly.
What is your participant role? KWOC

**Comment by abhisheks008** [comment]
Try to use different machine learning algorithms (at least 3-4) and compare them according to their accuracy scores to find out the best fitted one. Only using tensorflow and keras will not satisfy me!

@Han9128 issue assigned to you.
